apps/frontend/components/Card/card.tsx

```diff
@@ -59,9 +59,17 @@ function BulletPoint() {
   return <span>&nbsp;&bull;&nbsp;</span>;
 }
 
-function CardContent({ article }: { article: ArticleEntity }) {
-  const MAX_CHARACTERS = 1000;
-  const content = article.attributes.content.substring(0, MAX_CHARACTERS);
+function CardContent({ article, thumbnail }: { article: ArticleEntity, thumbnail: boolean }) {
+  let content = article.attributes.content;
+  const THUMBNAIL_MAX_CHARACTERS = 500;
+
+  if(thumbnail) {
+    if(content.length > THUMBNAIL_MAX_CHARACTERS) {
+      content = content.substring(0, THUMBNAIL_MAX_CHARACTERS) + 
+      `
+      \n ... czytaj dalej`;
```

Comment:
można to zmienić na coś ładniej ostylowanego + dodać link
![image](https://user-images.githubusercontent.com/61631665/175066991-4bf3ff8b-7f6a-4c7a-aa81-22e48a8431a9.png)